polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -19,6 +19,27 @@ enum Status {
 
 export const STYLE_CLASSES = ['one', 'two', 'three', 'four', 'five'];
 
+/**
+ * Computes a rudimentary hash from a string by xoring the character codes
+ * of all characters
+ * @param str A string
```

Comment:
```suggestion
 * @param {String}
```

Reply:
Or something like: `@param {String} $name - The avatar name.`

Reply:
The `@param` was added through autocompletion, so I left it without thinking. I don't think we even need it. Especially since this is already TS?